.github/workflows/test.yml

```diff
@@ -26,5 +26,8 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Linting
```

Comment:
This should be after the `npm install` step to fix the `run-s` errors.
Also in CI actions it's better to use `npm ci` to have reproducible builds (also it's faster than `npm install`).